Cyan/interpreter.py

```diff
@@ -293,9 +304,15 @@ def build_in_out(*values):
     return RTResult().success(NoneObj())
 
 
-def build_in_inp():
-    inp = String(input())
-    return RTResult().success(inp)
+def builtin_inp():
+    try:
+        inp = input()
+    except KeyboardInterrupt:
+        return RTResult()
```

Comment:
That doesn't make sense, it would make the inp() function return python `None`.. make it `return RTResult().success(NoneObj())` so it's a cyan `none`

Reply:
oh yea mb, fixed